ypkg2/main.py

```diff
@@ -190,6 +191,41 @@ def execute_step(context, step, step_n, work_dir):
             return False
     return True
 
+was_avx2_context = False
+
+def post_execute_step(context, step, step_n, work_dir):
+
+    hwcaps_v3_bin_dir = os.path.join(context.get_install_dir(), "usr", "hwcaps", "x86-64-v3", "bin")
+    hwcaps_v1_bin_dir = os.path.join(context.get_install_dir(), "usr", "hwcaps", "x86-64-v1", "bin")
+    bin_dir = os.path.join(context.get_install_dir(), "usr", "bin")
+
+    if context.avx2 and step_n == "install":
```

Comment:
Paired with my other suggestion about moving the step check, could we inverse this check and early return if the context is not `avx2`?